prisma/schema.prisma

```diff
@@ -89,3 +92,16 @@ model Post {
     createdAt          DateTime      @default(now())
     updatedAt          DateTime      @updatedAt
 }
+
+model Vote {
+  id               Int          @id @default(autoincrement())
+  user             User         @relation(fields: [userId], references: [id])
+  userId           String
+  transcription    Transcription @relation(fields: [transcriptionId], references: [id])
```

Comment:
I don't know prisma, sorry. What does this mean, and how does it translate to SQL?

Reply:
Prisma is a modern ORM (Object-Relational Mapping) framework that can assist developers in managing and operating databases more conveniently. When using Prisma, it automatically converts your Prisma model file into the corresponding SQL commands.

Once you have defined a Prisma model, Prisma will automatically generate the appropriate SQL commands for you.

Reply:
Thanks. So what's the difference between `transcription` and `transcriptionId` in this model?

Reply:
In the above data model, "Transcription" and "transcriptionId" are associated with each other. "Transcription" is a relationship field that points to the "Transcription" object in another model, while "transcriptionId" is a foreign key that stores the unique identifier of the "Transcription" object associated with the "Sentence" object.

Simply put, "Transcription" is a relationship field that allows access to the associated "Transcription" object and its properties. "transcriptionId", on the other hand, is a regular field that simply stores the unique identifier of the associated object. Typically, ORM frameworks use "transcriptionId" for querying and association operations.